toolz/itertoolz.py

```diff
@@ -214,7 +214,7 @@ def _merge_sorted_binary_key(seqs, key):
         yield val1
 
 
-def interleave(seqs, pass_exceptions=()):
+def interleave(seqs):
```

Comment:
@mrocklin, care to chime in?  I believe `pass_exceptions=()` was included from the very beginning when you consolidated your "utils.py" files from various projects. I know `logpy`, for instance, uses `pass_exceptions` in the `interleave` function it defines.

I agree that `pass_exceptions` seems out of place and not sufficiently justified, but @mrocklin has needed it in the past, and I'm generally in favor of maintaining backwards compatibility.